third_party/terraform/tests/resource_monitoring_uptime_check_config_test.go

```diff
@@ -39,7 +40,49 @@ func TestAccMonitoringUptimeCheckConfig_update(t *testing.T) {
 	})
 }
 
-func testAccMonitoringUptimeCheckConfig_update(path, project, pwd string) string {
+// The second update should force a recreation of the uptime check because 'monitored_resource' isn't
+// updatable in place
```

Comment:
We still don't have a way to test force-recreates, right?

Reply:
Sadly no :(